src/components/panels/SpindleSpeedPanel.vue

```diff
@@ -1,41 +1,55 @@
 <template>
-    <v-card>
-        <v-card-title><v-icon>mdi-45hammer-screwdriver</v-icon> Spindles</v-card-title>
-        <v-card-text>
-            <v-simple-table>
-                <thead>
-                    <th>Spindle Name</th>
-                    <th>Active</th>
-                    <th>Current RPM</th>
-					<th>Set RPM</th>
-                </thead>
-                <tbody>
-                    <tr v-for="(spindle, index) in spindles" :key="index" :class="{'spindle-active' : spindle.current > 0 }">
-                        <td>{{getName(spindle)}}</td>
-                        <td>
-								<v-btn v-if="spindle.current <= 0"  :value="1"  block @click="spindleOn(spindle, index)">On</v-btn>
-								<v-btn v-else :value="0" block @click="spindleOff(index)">Off</v-btn>
+	<v-card>
+		<v-card-title>
+			<v-icon>mdi-45hammer-screwdriver</v-icon>Spindles
```

Comment:
Please add a translation for this one too

Reply:
Updated translations